packages/assets-controllers/src/NftController.ts

```diff
@@ -164,55 +157,58 @@ export interface NftMetadata {
   attributes?: Attributes;
   lastSale?: LastSale;
   rarityRank?: string;
-}
+};
 
 /**
  * @type NftConfig
  *
  * NFT controller configuration
  * @property selectedAddress - Vault selected address
  */
-// This interface was created before this ESLint rule was added.
-// Convert to a `type` in a future major version.
-// eslint-disable-next-line @typescript-eslint/consistent-type-definitions
-export interface NftConfig extends BaseConfig {
+export type NftConfig = {
   selectedAddress: string;
   chainId: Hex;
   ipfsGateway: string;
   openSeaEnabled: boolean;
   useIPFSSubdomains: boolean;
```

Comment:
Fix for consistent capitalization?

```suggestion
  useIpfsSubdomains: boolean;
```